### DISCUSSION TITLE
Suggestions for Settings

### DISCUSSION THREAD

**Discussion by AykutSarac** [AykutSarac/jsoncrack.com#63] [Ideas]
Hi there!

I'm currently building a settings **modal** which will have bunch of **customizable options** for JSON Visio for your usage. I need ideas on what options to list there. Any idea is appreciated much!

Images below if you wanna see how it will look like :)

<img width="149" alt="image" src="https://user-images.githubusercontent.com/47941171/174318259-8f50222b-4b00-4e7a-85ac-a288a398153f.png">

<img width="510" alt="image" src="https://user-images.githubusercontent.com/47941171/174318282-c7fd6743-0057-4add-a858-2e6b1da4b6a9.png">

_w/ Love!
Aykut_

**Comment by FinnWinchester** [comment]
Well, for starters, would be nice if we could differenciate between strings and integers. The current example shows 2016 and any other string in white as if they were the same type and would be nice if we could:
a) Have different colors for different types.
b) [Nice to have] A way to customise said colors.

Thank you! Great tool!
